Kitodo-API/src/main/java/org/kitodo/api/dataformat/Workpiece.java

```diff
@@ -46,9 +46,15 @@
      */
     private Structure structure = new Structure();
 
+    /**
+     * The list of parent linked structures. The list is to be read from top to
+     * bottom, i.e. the element 0 of the list is the topmost structure.
+     */
+    private List<LinkedStructure> uplinks = new ArrayList<>();
```

Comment:
Do you really want to show this structure in an ArrayList? I think an own class, which is more specific is appropriate here